Makefile

```diff
@@ -48,6 +48,9 @@ test-unit: ## Run the unit tests
 verify: tidy generate format
 	git diff --exit-code
 
+install: generate
+	kubectl apply -f manifests
```

Comment:
does this install a single provisioner? all provisioners?

Reply:
I think in the current landscape it would be the CRDs + the requisite static manifests to install the k8s bundle provisioner? And in the future, package provisioners individually outside of the root manifest directory?

Reply:
if the provisioner can run locally too, this could be something like:

```
go run ./provisioner/k8s ./deploy
```

Reply:
Yeah that's an interesting take - might be worth just taking that route for the time being until we get something more substantial (e.g. an actual k8s provisioner deployment that gets installed, which is not present in these PR changes). Should be an easy addition the nested k8s/provisioner/README installation instructions once those gets added.

Reply:
Now that the manifests directory has moved, I think we need to either point directly to `provisioner/k8s/manifests` here or add a symlink to it in the top-level `manifests` directory.

Reply:
> go run ./provisioner/k8s ./deploy

@ecordell I like this, and I think it implies that `./deploy` is a directory containing a "plain k8s bundle".

Seems like a solid feature-request and long term direction for this.

Reply:
Let's just point directly to that directory path for now and iterate on the structure as we get more of this project built out.